### ISSUE TITLE
Unknown transaction type: 9 - solved

### ISSUE THREAD

**Issue by TheGoldenEye** [LiskHQ/lisk#18]
This issue references the issue #4, which was closed.
After compiling the new lisk-cli, and recreating the genesisBlock.json I still get the error:

$ node app.js 
log 2016-03-23 22:39:44 Lisk started: 0.0.0.0:7000 
error 2016-03-23 22:39:52 saveBlock Unknown transaction type: 9
fatal 2016-03-23 22:39:52 [Unknown transaction type: 9]

The reason is the created transaction in the geneses with the wrong transaction type 9.
If I change the type there manually to "type": 5, everything seems to work

**Comment by TheGoldenEye** [comment]
#4

**Comment by karmacoma** [comment]
@TheGoldenEye Please ensure you have pulled the latest source for both lisk and lisk-cli. Currently we are undergoing some changes to the core, so some compatibility has been broken here and there. I've been working to resolve this over the past few days. Should all be working now.

I have tried running this command myself and all is working fine here.

These commits should have resolved the issue:

- https://github.com/LiskHQ/lisk-cli/commit/2e8156bfe49b950432649bc6059d49f751aa1ec6
- https://github.com/LiskHQ/lisk-cli/commit/536c1eca99bd2a3944bb9df4329e1d4789de2458

If you could submit any further issues you are having with lisk-cli it would be much appreciated.

Please submit here: https://github.com/LiskHQ/lisk-cli/issues

Many thanks, Oliver.

**Comment by TheGoldenEye** [comment]
Hi Oliver,
I repeated
```
git pull
sudo npm -g install
```
with no success. :-(
Just to be sure that I run the correct program, i changed the version number in index.js to "1.1.2". (Should it be 1.1.3 ?)
```
sudo npm -g install
...
$ lisk-cli -V
1.1.2
```
Now I repeated to create the genesis - surprise - it works.
I do not know what I have done differently, perhaps the version-up has helped???
No matter, now it works.
Thank you for your support, oliver!
